packages/client/src/index.ts

```diff
@@ -1,10 +1,39 @@
-export { store } from './store';
+/**
+ * Internal dependencies
+ */
+import { store } from './store';
+import {
+	registerFeature,
+	unregisterFeature,
+	executeFeature,
+	getRegisteredFeature,
+	getRegisteredFeatures,
+} from './api';
+
+const publicApi = {
+	store,
+	registerFeature,
+	unregisterFeature,
+	executeFeature,
+	getRegisteredFeature,
+	getRegisteredFeatures,
+};
+
+if ( typeof window !== 'undefined' ) {
+	// @ts-ignore
+	window.wp = window.wp || {};
+	// @ts-ignore
+	window.wp.features = publicApi;
+}
```

Comment:
I don't really understand why you're setting the global variable manually. In Gutenberg, webpack does that automatically when we define the "external", the package itself should stay just like any regular npm package exporting stuff.